source/manage/logging.rst

```diff
@@ -167,6 +167,8 @@ Configuring advanced logging includes the following steps:
 Define advanced log output
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+Ensure that :ref:`audit logging is enabled <configure/experimental-configuration-settings:enable audit logging>` and that the :ref:`file name <configure/experimental-configuration-settings:file name>` is set to ``./logs/audit.log`` in the ``ExperimentalAuditSettings`` section of the ``config.json`` file. 
```

Comment:
That doesn't seem correct. `ExperimentalAuditSettings.AdvancedLoggingJSON` is a replacement for the stock audit setting. If it is necessary to enable `ExperimentalAuditSettings.FileEnabled`, it's a bug, right?

Reply:
Thanks, @hanzei. I may have misinterpreted the [context](https://hub.mattermost.com/private-core/pl/g1m56ra8tprd9n55nj6gnwq3yy) provided for this update:

"Logging didn’t kick in until we manually added the following settings to the config.json:

```
"ExperimentalAuditSettings": {
    "FileEnabled": true,
    "Filename": "./logs/audit.log"
}
```

Once these were in place and the service was restarted, everything started working as expected.

I interpret this as needing these 2 configuration settings set. Instead, is it the case that this text needs to be part of the JSON code itself? Appreciate your help with this!

Reply:
@cwarnermm I think the original context is not correct. Responded via https://hub.mattermost.com/private-core/pl/6k71ac971pfd3gbnyeozhf6u1r.

Reply:
Thanks! I'm following the thread discussion. Curious to know whether ``FileEnabled`` would be required in default cases since it's default is ``false``?